src/components/WeatherWidget/WeatherWidget.tsx

```diff
@@ -126,7 +139,8 @@ export default function WeatherWidget({ location }: Props) {
                     </Tooltip>
                     {`Temperature: ${roundTemperature(
```

Comment:
Let's create a function that receives a number with temparature and returns pretty-printed value